src/routes/api/articles/reportArticles.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable require-jsdoc */
```

Comment:
Kindly remove this comment. You may need to set that rule in the eslint configuration file.

Reply:
js-doc descriptions have been added to the file